pom.xml

```diff
@@ -231,6 +246,9 @@
           <argLine>
             ${surefireArgLine}
           </argLine>
+          <!-- fork a new JVM to isolate test suites, especially import with singletons -->
+          <forkCount>${cpu.count}</forkCount>
+          <reuseForks>false</reuseForks>
```

Comment:
This isn't directly related to my change, but just addresses a nasty testing issue.

Because we have a global singleton, many tests could interfere with eachother if they interleave. For example, 1 test might run `api.clearHooks()` immediately after another one does a setup that adds a hook.

This change runs every suite in a forked JVM, which adds a bit of startup time, but gives them each their own classloader to prevent interference via the singletons.